### PR TITLE
when error is detected in contexbroker connection, an user logged must be stay logged

### PR DESCRIPTION
**Pull request by jesuspg**
@jframos

### CODE REVIEW

File: dashboard/lib/routes/index.js [hunk 1]
```diff
@@ -55,11 +47,17 @@ function getIndex(req, res) {
...
 
         if (userinfo !== undefined) {
-
             //search for subscription
 
             logger.debug({op: 'index#get'}, 'Regions: %s', regions.constructor.name);
 
+            if (regions.length === 0) {
+                res.render('error', {name: userinfo.displayName, role: req.session.role,
+                    message: 'The connection to Context Broker has timed out or the service is down',
```
Comment (flopezag) [inline dashboard/lib/routes/index.js]: Why not the following message?

"Page Cannot be Displayed due to a Context Broker error (connection timed out or service was down)."

File: dashboard/lib/routes/index.js [hunk 2]
```diff
@@ -75,6 +73,12 @@ function getIndex(req, res) {
 
 
         } else {
+            if (regions.length === 0) {
+                res.render('error', {name: 'sign in', role: req.session.role, logoutUrl: config.idm.logoutURL,
+                    message: 'The connection to Context Broker has timed out or the service is down',
```
Comment (flopezag) [inline dashboard/lib/routes/index.js]: Idem

**Comment by flopezag** [comment]
LGTM

**Comment by jframos** [comment]
LGTM

**Pull request merged**